conf/messages.en

```diff
@@ -412,6 +412,7 @@ checkYourAnswers.detailsOfTheGoods.category = Type of goods
 checkYourAnswers.detailsOfTheGoods.quantity = Number of items
 checkYourAnswers.detailsOfTheGoods.vatRate = VAT rate
 checkYourAnswers.detailsOfTheGoods.country = Country
+checkYourAnswers.detailsOfTheGoods.producedInEu = Produced in EU
```

Comment:
going forward we don't need to keep the above `checkYourAnswers.detailsOfTheGoods.country` key, i think

Reply:
country is still used for export